From aca502d15f27dcdc962cf8a408b58535c137bc6e Mon Sep 17 00:00:00 2001
From: berbascum <berbascum@ticv.cat>
Date: Sun, 1 Sep 2024 15:39:57 +0200
Subject: [PATCH 4/6] Revert "(compiler) redefined-macro: hide warnings by
 adding -Wno-macro-redefined to KBUILD_CFLAGS in Makefile"

This reverts commit b3965cab5f37ecc710d7ed541fbb03fc6f0f9847.
---
 Makefile | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index d74ee7424cd3..2f8fab2496ce 100644
--- a/Makefile
+++ b/Makefile
@@ -429,8 +429,7 @@ KBUILD_CFLAGS   := -Wall -Wundef -Wstrict-prototypes -Wno-trigraphs -pipe \
 		   -Wno-unused-variable \
 		   -Wno-unused-label \
 		   -Wno-unused-function \
-		   -Wdeclaration-after-statement \
-		   -Wno-macro-redefined
+		   -Wdeclaration-after-statement
 
 ifeq ($(TARGET_BOARD_TYPE),auto)
 KBUILD_CFLAGS    += -DCONFIG_PLATFORM_AUTO
-- 
2.39.2

